### DISCUSSION TITLE
Welcome to hpm_sdk Discussions!

### DISCUSSION THREAD

**Discussion by jhqian** [hpmicro/hpm_sdk#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sadkotheguest** [comment]
Could you help with the PWM counter synchronisation using the SYNCI signal (https://github.com/hpmicro/hpm_sdk/discussions/23)?

**Comment by sadkotheguest** [comment]
@jhqian , hello!
Do you have any ideas how to realize synchronous parallel bus for HPM62xx?
(according to https://github.com/hpmicro/hpm_sdk/discussions/37)

**Reply by jhqian** [reply]
Could you please provide more information about your scenario? Like what kind of device will be connected to this parallel bus?

**Reply by sadkotheguest** [reply]
I want to connect an external DAC with parallel bus to MCU & I need to set the data on a parallel bus with a fixed frequency continuously.

**Reply by jhqian** [reply]
running at 20MHz right? how about the bus width 8bit or more? what's the address/data single mode? muxed?

**Reply by sadkotheguest** [reply]
Yes, the frequency is about 20 MHz. Bus width is 8 bit. 
There aren't any address lines - only 8-bit data lines and clock.

**Reply by jhqian** [reply]
so host mcu will only write 8-bit width data to that parallel device at 20Mhz for certain period of time. mcu will not read from that device or be notified by that device (interrupt), correct?

**Reply by sadkotheguest** [reply]
Yes, MCU will only write 8-bit width data to external device with synchronous clock signal. 
And I don't need any interrupts.

**Reply by jhqian** [reply]
it seems can be achieved as following:
- get 8 gpios from a same fgpio port
- use one gptmr channel to generate clock
- use one gptmr cmp to generate event to trigger DMA to move data from ram to gpio port (timing should be checked according to DAC datasheet)

**Reply by sadkotheguest** [reply]
Ok, thank you for advices.
